<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>1021</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QWidget" name="home_page">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>991</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
      <property name="title">
       <string>Welcome to News Management System</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>260</y>
         <width>531</width>
         <height>94</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;
font-size: 30px;</string>
          </property>
          <property name="text">
           <string>Already have an account?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="login_btn_home">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;
font-size: 27px;
background-color: #03045E;
color: white;</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>110</y>
         <width>531</width>
         <height>94</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;
font-size: 30px;</string>
          </property>
          <property name="text">
           <string>Don't you have an account?</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="register_btn_home">
          <property name="styleSheet">
           <string notr="true">font-weight: bold;
font-size: 27px;
background-color: #03045E;
color: white;</string>
          </property>
          <property name="text">
           <string>Sign up</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="register_page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>911</width>
        <height>591</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>621</width>
         <height>521</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLineEdit" name="nameinput_register">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter Your Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameinput_register">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter Your username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailinput_register">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter Your email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordinput_register">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter Your Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="secretcode_input_register">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter Admin Secret Code</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Account Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="typeacc_combo_register">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Admin</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QPushButton" name="registerbtn_registerpage">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>sign up</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="backbtn_registerpage">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>back</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="login_page">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1061</width>
        <height>591</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>651</width>
         <height>511</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Account Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="typeacc_combo_login">
            <item>
             <property name="text">
              <string>Admin</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QCommandLinkButton" name="forgetpassword_commandLink_login">
          <property name="text">
           <string>Forget Password</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="username_login_input">
          <property name="placeholderText">
           <string>Enter your username</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="login_btn_loginpage">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="back_btn_loginpage">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="password_login_input">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter Your Password</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="secretcode_input_login">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter Secret Code</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="forget_password">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1091</width>
        <height>651</height>
       </rect>
      </property>
      <property name="title">
       <string>Forget Password</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>981</width>
         <height>561</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="usernameinput_FP">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailinput_FP">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter Email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="secretkey_FP">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter secret key</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Account Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="acctype_FP">
            <item>
             <property name="text">
              <string>Admin</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="resetpassword_btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="text">
           <string>Reset Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="newpassword_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>981</width>
             <height>231</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLineEdit" name="newpasswordinput_FP">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Enter New Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="confirnnewpasswordinput_FP">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>Enter New Password again</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="confirmbtn_reset">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Confirm</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backbtn_FP">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
